sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/TPCDSQueryBenchmark.scala

```diff
@@ -99,6 +95,20 @@ object TPCDSQueryBenchmark {
   }
 
   def main(args: Array[String]): Unit = {
+    if (args.length < 1) {
```

Comment:
Could we also allow another way to run this benchmark?

We can hardcode the value of `dataLocation` and run it in IntelliJ directly.

Reply:
@sarutak kindly ping

Reply:
We can pass the argument through the run-configuration even though we use IDE like IntelliJ right?
Or, how about give `dataLocation` through a new property?

Reply:
@sarutak @maropu Could we do something like https://github.com/apache/spark/blob/master/resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/ApplicationMasterArguments.scala? 

We also can use add another option for outputing the plans of TPC-DS queries, instead of running all the queries.

Reply:
Good idea. I'll add `TPCDSQueryBenchmarkArguments`.